### DISCUSSION TITLE
Add "right click to go to specific part of map select"

### DISCUSSION THREAD

**Discussion by EWReinE** [ppy/osu#30681] [Ideas] [answered]
So in stable, something I do a lot is use right click in menu to instantly go to a specific part of the map scroll instead of scrolling with mms over a dozen times. Its something I do a lot and it would be more convenient than the current way its done in lazer(left click scroll wheel -> move scroll wheel). 

below are 2 videos showing how it works in stable and lazer;
https://drive.google.com/drive/folders/1nBJbgqgdtpww3Q9QaVeskI3fw08WIv4Z?usp=drive_link

I don't think its a bad thing to have a click+move scroll bar but if you have enough maps, sometimes its annoying to click on the bar because you miss!(osu player misses lol) However, you could add an area that will detect a right click in the empty space between the info menu on the left and the map scroll on the right where you can right click and move the scroll bar to where ever the cursor is (up and down. no left to right movement.) similar to how it behaves in stable.

If this could be added, I think it would make the lazer experience much smoother when scrolling through to a specific section.

**Comment by Joehuu** [comment, marked as answer]
Drive link is access denied, but the setting you're looking for is:

![osu!_MjYjPxOBwS](https://github.com/user-attachments/assets/f06ebbaa-3551-40b4-bd6b-d8edb755e7d3)

**Comment by EWReinE** [comment]
ok thanks.
